lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
     return `did:key:${key.fingerprint()}#${key.fingerprint()}`;
   }
+
+  /**
+   * Generates a public verification method from a keypair.
+   *
+   * @param {LDKeyPair} key
+   *
+   * @returns {string} Returns the key's id.
+   */
+  getPublicVerificationMethod({key}) {
```

Comment:
Do we need "Public" here? Also, why is this a function the driver? Shouldn't this be some kind of `export` function on the key pair instead?

Reply:
Possibly, but then we'd have to change the interface for LD key pairs, right? Again, attempting to avoid major surgery -- I can make it a function for now?